spark/src/test/scala/org/apache/comet/CometCastSuite.scala

```diff
@@ -66,19 +66,22 @@ class CometCastSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     castTest(testValues, DataTypes.BooleanType)
   }
 
-  ignore("cast string to byte") {
-    castTest(generateStrings(numericPattern, 8).toDF("a"), DataTypes.ByteType)
+  test("cast string to byte") {
+    val testValues =
+      Seq("", ".", "0", "-0", "+1", "-1", ".2", "-.2", "1e1", "127", "128", "-128", "-129") ++
+        generateStrings(numericPattern, 8)
+    castTest(testValues.toDF("a"), DataTypes.ByteType)
   }
 
-  ignore("cast string to short") {
+  test("cast string to short") {
```

Comment:
We should probably have some negative tests with invalid strings.
Also, curious, what does `cast(".")` yield?

Reply:
The fuzz testing does generate many invalid inputs. I can add some more explicit ones to these tests, though.

`cast(".")` will yield different results depending on the eval mode:

- `LEGACY` -> `0`
- `TRY` -> null
- `ANSI` -> error

Reply:
+1 for these test btw